vep_mapping_pipeline/run_consequence_mapping.sh

```diff
@@ -1,11 +1,17 @@
 #!/bin/bash
+set -o pipefail
```

Comment:
may be add -x and -e too? see https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

Reply:
Yes, very good idea, will do. Even though the script is small and simple, those safety options should really be the default.

Reply:
Addressed in https://github.com/EBIvariation/vep-mapping-pipeline/pull/2/commits/f115843035ba53aaa7ff6e0905e64df541dadf42